cmd/kusk/Makefile

```diff
@@ -59,8 +59,8 @@ test:
 
 .PHONY: install-deps
 install-deps:
-	go install github.com/go-bindata/go-bindata/v3/go-bindata@v3.1.3
-	go install sigs.k8s.io/kustomize/kustomize/v4@v4.5.2
+	@type go-bindata >/dev/null 2>&1 || go install github.com/go-bindata/go-bindata/v3/go-bindata@v3.1.3
```

Comment:
What's `@type`? Are we checking if it's already installed?

Reply:
If `go-bindata` is not installed, it will install it, basically. It's sort of like `command -v <some_command>`. Here's a good example of how to use it: https://stackoverflow.com/a/677212/16361002.